packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -3,20 +3,581 @@ import type { Plugin } from "graphile-build";
 import withPgClient from "../withPgClient";
 import { parseTags } from "../utils";
 import { readFile as rawReadFile } from "fs";
-import pg from "pg";
 import debugFactory from "debug";
 import chalk from "chalk";
 import throttle from "lodash/throttle";
 import flatMap from "lodash/flatMap";
-import { quacksLikePgPool } from "../withPgClient";
-
 import { version } from "../../package.json";
+import pg from "pg/lib/index";
```

Comment:
Why not just `pg`? This seems brittle; it's generally a bad idea to require from the internals of a package unless they've deemed doing so supported.

Reply:
This was a mistaken auto import by my IDE. I'll fix it.

Reply:
This is not yet fixed